FirebaseSessions/Tests/Unit/SessionStartEventTests.swift

```diff
@@ -15,7 +15,15 @@
 
 import XCTest
 
-@_implementationOnly import GoogleUtilities
+#if SWIFT_PACKAGE
+  import FirebaseSessionsObjC
+#endif // SWIFT_PACKAGE
+
+#if SWIFT_PACKAGE
+  import GoogleUtilities_Environment
+#else
+  import GoogleUtilities
+#endif // SWIFT_PACKAGE
```

Comment:
The imports for SPM are a little clunky - I wonder if there's a cleaner way to do this

Reply:
Can we extract some of these common inclusions to a separate file and import that? Would be helpful to centralize this clunky code?

Reply:
I don't think that is feasible. Swift doesn't have macros-- something that would help here.